1a.cpp

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <vector> 
+using namespace std;
+
+int main()
+{
+	vector<int> v1(5);
```

Comment:
The way you use `std::vector` here is not the way it was designed to be used. You might as well just use `std::array` since you are not allocating any additional memory during the runtime.